app/Http/Controllers/CategoryController.php

```diff
@@ -45,7 +59,15 @@ public function store(Request $request)
      */
     public function show($id)
     {
-        //
+        $category = Category::findOrFail($id);
+
+        if ($category->parent_id == 0) {
+            $category->load('children');
+
+            return view('website.backend.category.show', ['category' => $category]);
```

Comment:
```suggestion
            return view('website.backend.category.show', compact('category'));
```
mấy cái này c thấy tiện hơn c suggest thôi nhé, k phai em dùng sai đâu